File: Assignmentno19/task_4.java
import java.util.*;

class Logic 
{
    void Display(int ino)
    {
        int Digit = 0;
        while(ino>0)
        {
            Digit = ino%10;
            System.out.println("The Seperated Number will be:"+Digit);
            ino=ino/10;
        }
      
        
    }
}

class task_4
{
    public static void main(String C[])
    {
        int iNo;
        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();


        System.out.println("Enter the Number:");
        iNo = sobj.nextInt();
        lobj.Display(iNo);



    }
}

File: Assignmentno69/task_4.c
#include<stdio.h>

void Display(int iNo)
{
  
   static char i = 'A';
   
   if(iNo == 0)
   {
     return;
     
   }
    printf("%c\t",i);
    i++;
    Display(iNo-1);
  

}

int main()
{
    int iValue = 0;
    printf("Enter the Number:\n");
    scanf("%d",&iValue);
    Display(iValue);

    return 0;
}